README.md

```diff
@@ -6,13 +6,28 @@ This is my educational project within IT KPI Python mentorship program by @webkn
 
 ## Installation
 
-Soon
+1. Clone this repository:
+```console
```

Comment:
For the code block and the text paragraph to be a part of the numbered list, indent it, so it starts right under the word “Clone”.

Reply:
Addressed https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/19/commits/6791bbf4a38719468006c4991bf5b12ef6cfb08d

Reply:
Marking this thread as resolved.